<launch>

  <arg name="image_raw_topic" default="/ackermann_vehicle/camera/rgb/image_raw" />
  <arg name="twist_cmd_topic" default="/cmd_vel" />
  <arg name="twist_linear_x" default="1" />
  <arg name="signal_cmd_topic" default=""/>
  <arg name="vel_cmd_topic" default="" /> <!-- /android_input_vel -->
  <arg name="model" default=""/>
  <arg name="urdf" default="robot05_040_038"/>


  <group if="$(eval arg('model') != '')">
    <node name="driving" pkg="cnn" type="driving1.py" output="screen" required="true" >
      <param name="image_raw_topic" value="$(arg image_raw_topic)" />
      <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)" />
      <param name="twist_linear_x" value="$(arg twist_linear_x)" />
      <param name="vel_cmd_topic" value="$(arg vel_cmd_topic)" />
      <param name="signal_cmd_topic" value="$(arg signal_cmd_topic)"/>
      <param name="modelname" value="$(arg model)" />
      <param name="urdf" value="$(arg urdf)" />
    </node>
  </group>
  
</launch>
